openpype/hosts/hiero/plugins/publish/extract_frames.py

```diff
@@ -36,7 +36,7 @@ def process(self, instance):
             output_path = output_template
             output_path += ".{:04d}.{}".format(int(frame), output_ext)
 
-            args = [oiio_tool_path]
+            args = list(oiio_tool_args)
```

Comment:
isn't this list anyway, as `get_oiio_tools_args()` always returns list?

Reply:
The `args` list is modified through the loop. It would keep arguments from previous loop on next run.